pkgs/tools/security/beyond-identity/default.nix

```
@@ -0,0 +1,99 @@
+{ lib, stdenv, fetchurl, dpkg, buildFHSUserEnv
+, glibc, glib, openssl, tpm2-tss
+, gtk3, gnome, polkit, polkit_gnome
+}:
+
+let
+  version = "2.45.0-0";
+  name = "beyond-identity";
+
+  src = fetchurl {
+    url = "https://packages.beyondidentity.com/public/linux-authenticator/deb/ubuntu/pool/focal/main/b/be/${name}_${version}/${name}_${version}_amd64.deb";
+    sha512 = "852689d473b7538cdca60d264295f39972491b5505accad897fd924504189f0a6d8b6481cc0520ee762d4642e0f4fd664a03b5741f9ea513ec46ab16b05158f2";
+  };
+
+  libPath = lib.makeLibraryPath ([
+    glib glibc openssl tpm2-tss
+    gtk3 gnome.gnome-keyring
+    polkit polkit_gnome
+  ]);
+in let beyond-identity = stdenv.mkDerivation {
+  pname = name;
+
+  inherit version;
+  inherit src;
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [
+    dpkg
+  ];
+
+  installPhase = ''
+    dpkg -x $src .
+    mkdir -p $out/opt/beyond-identity
+
+    rm -rf usr/share/doc
+
+    # https://github.com/NixOS/nixpkgs/issues/42117
+    sed -i -e 's/auth_self/yes/g' usr/share/polkit-1/actions/com.beyondidentity.endpoint.stepup.policy
+
+    cp -arv usr/{bin,share} $out
+    cp -arv opt/beyond-identity/bin $out/opt/beyond-identity
```

```suggestion
    cp -ar usr/{bin,share} $out
    cp -ar opt/beyond-identity/bin $out/opt/beyond-identity
```
Lets not spam the build log with useless information.